lambda_functions/submission_service/submit_item.py

```diff
@@ -13,6 +14,9 @@
 logger.setLevel(logging.INFO)
 
 
+def remove_control_characters(s):
+    return "".join(ch for ch in s if unicodedata.category(ch)[0]!="C")
```

Comment:
If I understand this correctly, this line removes all control characters like line breaks, right? 
This would make longer passages hard to read in the review process/archive. 
Maybe we can find a better way of dealing with this issue without compromising in terms of readability? Or maybe change the encoding in the frontend?

Reply:
At the moment control characters are only removed from the step function input, the machine can still read it ;-)

Reply:
Ah, right. The item is saved to the db before the stepfunction is invoked. Then let's see if this works :)